### PR TITLE
avoid running several times the tests on Travis

### PR DESCRIPTION
**Pull request by monperrus**

### CODE REVIEW

**Comment by retuxx** [comment]
Finally :)

File: .travis.yml [hunk 1]
```diff
@@ -5,10 +5,20 @@ sudo: required
 jdk:
   - oraclejdk8
 
+# from https://docs.travis-ci.com/user/customizing-the-build
+# A build on Travis CI is made up of two steps: install: (install any dependencies required), script: (run the build script)
+
+install:
+  - # download deps with maven
+  - mvn dependency:resolve
```
Comment (GerardPaligot) [inline .travis.yml]: Why this is necessary?

Comment (monperrus) [inline .travis.yml]: because otherwise you have the default "mvn install"

Comment (GerardPaligot) [inline .travis.yml]: Ok. I got it.

File: .travis.yml [hunk 2]
```diff
@@ -5,10 +5,20 @@ sudo: required
...
+  
 script:
   # Verify includes checkstyle, license and animal sniffer.
-  - mvn verify javadoc:javadoc site
-  - mvn test jacoco:report coveralls:report -Pcoveralls --fail-never
+  - mvn verify javadoc:javadoc site -DskipTests
+  # the core of continuous integration
+  - mvn test jacoco:report
```
Comment (GerardPaligot) [inline .travis.yml]: Unfortunately, the code tested is the code instrumented by jacoco. So, you can found error like this one: https://travis-ci.org/INRIA/spoon/builds/161590324#L5188

**Pull request merged**